Bottom: 2ee44c4fb7057af37e9a41fc8bcc78e167a2d52a
Top:    abd028d5ef4922dc3b8eecce2036c5bd012710ff
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-03 14:34:05 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
